test/utils.ts

```diff
@@ -336,56 +348,71 @@ export const fakeTDecFlow = ({
 
     const decryptionShare = aggregate.createDecryptionShareSimple(
       dkg,
-      ciphertext.header,
-      aad,
+      thresholdMessageKit.ciphertextHeader,
+      authenticatedData.aad(),
```

Comment:
The authenticated data should never be used outside of encryption - it's only use is to ensure a consistent aad is used for encryption and decryption since the ACP can only be constructed after the data is encrypted (Ciphertext is produced).

```suggestion
      thresholdMessageKit.acp.aad(),
```